config/locales/en/en.yml

```diff
@@ -369,6 +369,15 @@ en:
     created: "Created"
     caption: "Caption"
     condense_all: "Condense All"
+    currency:
```

Comment:
Let's use one and other here to account for different amounts and to extend this translated functionality.

For example:
```
argentinean_peso:
  one: "Argentinean Peso"
  other: "Argentinean Pesos"
```

There are other parts of the system that use the currency list without translations. We'd like to provide support moving forward for the currency names to be translated into Spanish when a user is viewing any of the sites in Spanish.

For example, this is the loan details form. When selecting the currency and viewing the site in Spanish, the currencies are untranslated.

<img width="1049" alt="image" src="https://user-images.githubusercontent.com/1853878/43780886-a6d64cf6-9a29-11e8-9f8c-c180aac677e0.png">

Making changes to the loan details form is not needed in this ticket. We want these translations to work for both singular and plural currency names. That way we can do easy replacements as we see them moving forward (in different tickets).

Reply:
This was what I originally did but I figured that it was only used in that one place in the app. Didn't notice the other

Will update right away

Reply:
Thank you!